src/Halogen/Aff/Driver/Eval.purs

```diff
@@ -48,9 +47,7 @@ handleLifecycle lchs f = do
   result <- liftEff f
   { initializers, finalizers } <- liftEff $ readRef lchs
   forkAll finalizers
-  -- No need to par/fork initializers here as there's only ever zero or one at
-  -- this point, due to the squashing at each level of the component hierarchy.
-  sequence_ initializers
+  parSequence_ initializers
```

Comment:
Why is the comment no longer valid (and you want the initializers to be parallel)?